packages/apple-pay-integration/src/ApplePayPaymentMethod.tsx

```diff
@@ -0,0 +1,43 @@
+import { toResolvableComponent, PaymentMethodProps } from '@bigcommerce/checkout-js/payment-integration';
+import React, { FunctionComponent, useEffect } from 'react';
+
+const ApplePaymentMethod: FunctionComponent<PaymentMethodProps> = ({ method, checkoutService, language, onUnhandledError }) => {
+    useEffect(() => {
+        const componentDidMount = async ():Promise<void> => {
```

Comment:
You should be able to rename the function to something more related like `initializePayment`?

Reply:
Sure, I was going to ask about this. This naming is for nostalgia. 😂 So you can see two old friends.
Will rename them.